### PR TITLE
Added context to exception to ease debugging.

### PR DESCRIPTION
**Pull request by olvesh**
Added some context information to a possible matcher exception. This eased my debugging, hope it can help others.

### CODE REVIEW

File: collector/src/main/java/io/prometheus/jmx/JmxCollector.java [hunk 1]
```diff
@@ -259,15 +261,22 @@ public void recordBean(
...
-              if (lowercaseOutputLabelNames) {
-                labelName = labelName.toLowerCase();
-              }
-              if (!labelName.isEmpty() && !labelValue.isEmpty()) {
-                labelNames.add(labelName);
-                labelValues.add(labelValue);
-              }
+              final String unsafeLabelName = rule.labelNames.get(i);
+              final String labelValReplacement = rule.labelValues.get(i);
+                try {
```
Comment (brian-brazil) [inline collector/src/main/java/io/prometheus/jmx/JmxCollector.java]: The indentation is off here

**Comment by brian-brazil** [comment]
Thanks for the PR, what was the exact problem you were running into?

**Comment by olvesh** [comment]
Just when copy-pasting a lot of rules it was hard to spot where my labels and group references were off. This gave me the context to find them.

I did not look through all of the code to rework the error handling-  guess this could be applied several places with a broader try-catch.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**